tests/console/yast2_ntpclient.pm

```diff
@@ -14,14 +14,40 @@ use strict;
 use base "console_yasttest";
 use testapi;
 use utils qw(type_string_slow zypper_call);
+use version_utils qw(is_sle sle_version_at_least is_leap leap_version_at_least);
 
 sub run {
     select_console 'root-console';
-    # Test often fails due to info kernel messages disrupting screen
-    # Decrease logging level to warning to avoid this
-    assert_script_run "dmesg -n 4";
+
+    # ntp configuration is different in SLE15/Leap15/Tumbleweed
+    # use sle_or_leap_15 variable to avoid executing is_* and *_version_at_least multiple time
+    my $sle_or_leap_at_least_15 = ((is_sle && sle_version_at_least '15') || (is_leap && leap_version_at_least '15.0' || is_tumbleweed)) ? 1 : 0;
+
+    # define some keys used during ntp configuration
+    my %menu_keys = (
+        sync_interval       => 'alt-n',
```

Comment:
I guess we should better put in [cmd hash](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L37), and can redefine in case of sle15

Reply:
Looks better yes, I will modify this.